website_docs/_index.md

```diff
@@ -30,3 +30,162 @@ The current release can be found [here](https://github.com/open-telemetry/opente
 - [Quick Start](https://github.com/open-telemetry/opentelemetry-ruby#quick-start)
 - [API Documentation](https://open-telemetry.github.io/opentelemetry-ruby/)
 - [Examples](https://github.com/open-telemetry/opentelemetry-ruby/tree/main/examples)
+
+## Getting Started
+
+### Installation
+
+The API and SDK packages are available on RubyGems.org, and can be installed via `gem`:
+
+```sh
+gem install opentelemetry-api
+gem install opentelemetry-sdk
+```
+
+or via `Bundler` by adding the following to your `Gemfile`:
+
+```ruby
+gem 'opentelemetry-api'
+gem 'opentelemetry-sdk'
+```
+
+followed by:
+
+```sh
+bundle install
+```
+
+### Initialization and Configuration
+
+```ruby
+require 'opentelemetry/sdk'
+
+# Configure the sdk with default export and context propagation formats
+# see SDK#configure for customizing the setup
+OpenTelemetry::SDK.configure
+
+# To start a trace you need to get a Tracer from the TracerProvider
+tracer = OpenTelemetry.tracer_provider.tracer('my_app_or_gem', '0.1.0')
+```
+
+### Instrumentation
+
+#### Creating Spans
+
+```ruby
+  begin
+    span = tracer.start_span("foo", kind: :internal)
+    Trace.with_span(span) do |span, context|
+      # custom code here
+      pp "Hello World!"
+    end
+  rescue Exception => e
+    span&.record_exception(e)
+    span&.status = Status.error("Unhandled exception of type: #{e.class}")
+    raise e
+  ensure
+    span&.finish
+```
+
+```ruby
+# create a span
+tracer.in_span('foo') do |span|
+  # set an attribute
+  span.set_attribute('platform', 'osx')
+  # add an event
+  span.add_event('event in bar')
+  # create bar as child of foo
+  tracer.in_span('bar') do |child_span|
+    # inspect the span
+    pp child_span
+  end
+end
+```
+
+#### Attributes
+
+Attributes are keys and values that are applied as metadata to your spans and are useful for aggregating, filtering, and grouping traces. Attributes can be added at span creation, or at any other time during the lifecycle of a span before it has completed.
+
+```ruby
+# setting attributes at creation...
+tracer.in_span('foo', attributes: {  "hello" => "world", "some.number" => 1024, "tags" => [ "bugs", "won't fix" ] }, kind: :internal) do |span|
+
+  # ... and after creation
+  span.set_attribute("animals", ["elephant", "tiger"])
+
+  span.add_attributes({ "my.cool.attribute" => "a value", "my.first.name" => "Oscar" })
```

Comment:
It's worth pointing out in a comment that it is more efficient to use this to add multiple attributes in a single operation rather than `set_attribute`.